### DISCUSSION TITLE
Preview 1.20.10572.0 ?

### DISCUSSION THREAD

**Discussion by rexbinary** [microsoft/terminal#16770] [General]
The Microsoft Store just updated my Windows Terminal Preview to version 1.20.10572.0. I don't see that release here, and WSL won't load now either. Is this a valid release?

**Comment by DHowett** [comment]
It is a valid release. The release notes are scheduled to go out tomorrow. What symptoms are you experiencing with WSL?

**Comment by rexbinary** [comment]
WSL wasn't starting for some reason with a network error, but a reboot corrected it. Coincidence it seems. Thank you for confirming.

**Reply by DHowett** [reply]
Sure thing! Thanks!
